File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
1
2
3
4
5
6
7
8
9
10
11
12
13
function solution(price) {
    var answer = price;
    if (answer >= 500000) {
        answer *= 0.8;
    } else if (answer >= 300000) {
        answer *= 0.9;
    } else if (answer >= 100000) {
        answer *= 0.95;
    }

    return parseInt(answer);
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
def solution(price):
    if price>=500000:
        return int(price*0.8)
    
    elif price>=300000:
        return int(price*0.9)
    
    elif price>=100000:
        return int(price*0.95)
    else: #10만원 미만일 떄 (즉, 할인을 받지 않을 때)
        return price